src/frontends/ir/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@
 
 set(TARGET_NAME "ir_ov_frontend")
 
+if(CMAKE_COMPILER_IS_GNUCXX)
+    ie_add_compiler_flags(-Wno-missing-declarations)
+    ie_add_compiler_flags(-Wno-sign-compare)
+endif()
```

Comment:
why? I suppose we need to fix it, can we at least use `-Wno-error=` instead of `-Wno`?